frontend/src/libs/components/chat-sidebar/chat-sidebar.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback, useSearch, useState } from '../../hooks/hooks.js';
+import { Search } from '../components.js';
+import { ChatElement } from './chat-element/chat-element.js';
+import styles from './styles.module.scss';
+
+//mocked data
+const chats = [
+  { id: 1, name: 'Relationship' },
+  { id: 2, name: 'Friends' },
+  { id: 3, name: 'Family' },
+  { id: 4, name: 'Work' },
+];
+
+const ChatSidebar: React.FC = () => {
+  const [filter, setFilter] = useState('');
+  const [selectedChat, setSelectedChat] = useState({ id: 1 });
+  const handleFilterChange = useCallback(
+    (event: React.ChangeEvent<HTMLInputElement>): void => {
+      setFilter(event.target.value);
+    },
+    [],
+  );
```

Comment:
Let's use debounce function to reduce the number of searches

Reply:
done